### DISCUSSION TITLE
F=ma

### DISCUSSION THREAD

**Discussion by mingliuhermit** [BoomingTech/Piccolo#315] [Q&A] [answered]
void PhysicsSystem::integrateAccelerate(float delta_time)
    {
        for (auto& object_iter : m_physics_actors)
        {
            float inverse_mass = object_iter->getInverseMass();

            Vector3 linear_velocity = object_iter->getLinearVelocity();
            Vector3 force           = object_iter->getForce();
            Vector3 accel           = force * inverse_mass;

            if (m_is_use_gravity && inverse_mass > 0)
            {
                accel = accel + m_gravity;
            }
...
}
}
这里的Vector3 accel           = force * inverse_mass;是不是写错了，加速度不是应该等于F/m吗？

**Comment by ShenMian** [comment, marked as answer]
除以一个数, 等于乘以这个数的倒数. 此处 `inverse_mass` 估计就是 m 的倒数.

**Reply by mingliuhermit** [reply]
满脸羞愧，英语渣渣，只知道mass，没注意inverse
